app/src/main/res/raw/javascript_bridge.js

```diff
@@ -23,3 +23,6 @@ function reportOverScroll(clientWidth, scrollWidth, messageUid) {
 function reportError(error, scriptFirstLine, messageUid) {
     window.kmail.reportError(error.name, error.message, error.stack, scriptFirstLine, messageUid);
 }
+function webviewFinishedLoading() {
```

Comment:
```suggestion

function webviewFinishedLoading() {
```

Reply:
Is this really where you meant that comment to be?